.env.sample

```
@@ -0,0 +1,5 @@
+# RPC
+RPC_URL='REPLACE_WITH_YOUR_RPC_URL'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)